books/spiders/book.py

```diff
@@ -0,0 +1,69 @@
+import scrapy
+from scrapy.http import Response
+
+
+class BookSpider(scrapy.Spider):
+    name = "book"
+    allowed_domains = ["books.toscrape.com"]
+    start_urls = ["https://books.toscrape.com/"]
+
+    def parse(self, response: Response, **kwargs) -> None:
+        for product_url in response.css("h3 > a::attr(href)").getall():
+            product_url = response.urljoin(product_url)
+            yield scrapy.Request(product_url, callback=self.parse_product)
```

Comment:
The `parse` method correctly iterates over product URLs and makes requests to each product page using `scrapy.Request`. Ensure that the CSS selector `h3 > a::attr(href)` accurately targets the product links on the website.